common/views/themes/mixins.ts

```diff
@@ -11,7 +12,9 @@ export function respondBetween(
   maxBreakpoint: Breakpoint,
   content: string
 ): string {
-  return `@media (min-width: ${themeValues.sizes[minBreakpoint]}px) and (max-width: ${themeValues.sizes[maxBreakpoint]}px) {
+  return `@media (min-width: ${themeValues.sizes[minBreakpoint] ??
+    themeValues.tweakpoints[minBreakpoint]}px) and (max-width: ${themeValues
+    .sizes[maxBreakpoint] ?? themeValues.tweakpoints[maxBreakpoint] - 1}px) {
```

Comment:
I added `- 1` here – this was missing from when I originally converted the [mixins from scss](https://github.com/wellcomecollection/wellcomecollection.org/blob/convert-header-scss/common/styles/utilities/_mixins.scss#L19-L33)